python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -128,67 +155,125 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.usedResources).length > 0
+                  ? Object.entries(actor.usedResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : null,
+              priority: "primary",
             },
             {
               label: "Pending",
               value: actor.taskQueueLength.toLocaleString(),
+              priority: "primary",
             },
             {
               label: "Executed",
               value: actor.numExecutedTasks.toLocaleString(),
+              priority: "primary",
             },
             {
               label: "NumObjectIdsInScope",
               value: actor.numObjectIdsInScope.toLocaleString(),
+              priority: "secondary",
             },
             {
               label: "NumLocalObjects",
               value: actor.numLocalObjects.toLocaleString(),
+              priority: "secondary",
             },
             {
               label: "UsedLocalObjectMemory",
               value: actor.usedObjectStoreMemory.toLocaleString(),
+              priority: "secondary",
             },
-            // {
-            //   label: "Task",
-            //   value: actor.currentTaskFuncDesc.join(".")
-            // }
           ]
         : [
             {
               label: "ID",
               value: actor.actorId,
+              priority: "primary",
             },
             {
               label: "Required resources",
               value:
-                Object.entries(actor.requiredResources).length > 0 &&
-                Object.entries(actor.requiredResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.requiredResources).length > 0
+                  ? Object.entries(actor.requiredResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : null,
+              priority: "primary",
             },
           ];
 
+    // Apply transformation to add styling for information field
+    const transforms: Record<string, (info: ActorInformation) => ReactNode> = {
+      Pending: (info: ActorInformation) => {
+        if (actor.state !== -1 && actor.taskQueueLength >= 50) {
+          return (
+            <span
+              className={classes.warningTooManyPendingTask}
+              key={info.label}
+            >
+              {info.label}: {info.value}
+            </span>
+          );
+        } else {
+          return info.label;
```

Comment:
Shouldn't this be a `span` too so that we get the left margin for spacing? In fact, if all of these are `span`s, maybe we could try to enforce this in the typing using something like `JSX.Element` as the return type rather than the generic `ReactNode` (and then remove the latter from our imports).